Source/Core/VideoCommon/ConstantManager.h

```diff
@@ -8,6 +8,7 @@
 
 // all constant buffer attributes must be 16 bytes aligned, so this are the only allowed components:
 typedef float float4[4];
+typedef float float2[2];
```

Comment:
Totally missed the comment!  Argh.  Sorry

Why is there a restriction on 16byte alignment?